solr/core/src/java/org/apache/solr/util/hll/SerializationUtil.java

```diff
@@ -137,8 +137,8 @@ public static byte packVersionByte(final int schemaVersion, final int typeOrdina
    *     <ul>
    *       <li>If 'explicit-disabled' is chosen, this value should be <code>0</code>.
    *       <li>If 'auto' is chosen, this value should be <code>63</code>.
-   *       <li>If a cutoff of 2<sup>n</sup> is desired, for <code>0 <= n < 31</code>, this value
-   *           should be <code>n + 1</code>.
+   *       <li>If a cutoff of 2<sup>n</sup> is desired, for <code>{@code 0 <= n < 31}</code>, this
+   *           value should be <code>{@code n + 1}</code>.
```

Comment:
Duplicate code tags?

Reply:
so I went back and forth on this a bunch. I need to look at the actual generated javadoc to see what it looks like. Will check it.

Reply:
ok need to fix this based on https://docs.oracle.com/en/java/javase/19/docs/specs/javadoc/doc-comment-spec.html

Reply:
fixed in 74ffc2219930d742f38dacee4091d7624c0ddcaf